src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -95,6 +100,51 @@ public boolean isEnabled(final String toggleName, final UnleashContext context ,
         return enabled;
     }
 
+    @Override
+    public Variant getVariant(String toggleName, UnleashContext context) {
+        return getVariant(toggleName, context, "");
+    }
+
+    @Override
+    public Variant getVariant(String toggleName, UnleashContext context, String defaultPayload) {
+        final  FeatureToggle featureToggle = toggleRepository.getToggle(toggleName);
+        final boolean isEnabled = isEnabled(toggleName, context, false, featureToggle);
+
+        if(isEnabled) {
+            return selectVariant(featureToggle, context)
+                .map(variantDefinition -> new Variant(variantDefinition.getName(), variantDefinition.getPayload(), isEnabled))
+                .orElseGet(() -> new Variant("default", defaultPayload, isEnabled));
```

Comment:
The fallback needs to be stream-lined across all client implementations. 

In the code we should also hard-code a fallback-variant, no need to create  a new one every time. 
`Variant fallback = new Variant("disabled", false)`

Reply:
The payload default payload may differ. We should return the specified default payload on "disabled" right ?

Reply:
Fixed.